internal/bcdb/db.go

```diff
@@ -68,6 +68,9 @@ type DB interface {
 	// GetDBStatus returns status for database, checks whenever database was created
 	GetDBStatus(dbName string) (*types.GetDBStatusResponseEnvelope, error)
 
+	// GetDBIndex returns the defined index for a given database
```

Comment:
describe the expected format of the index and how to handle it, i.e. `json.Unmarshal` etc...

Reply:
Done